### ISSUE TITLE
Can translatable strings use nesting?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/build-an-atom#457]
I remember there was some discussion/progress about this earlier this year, but I cannot remember where we landed. Does our translation system support nested keys for translatable (non-a11y) keys?

I saw these in the yaml:
```yaml
keyboardHelpContent:
  particleNavigationHeading: Particle Navigation
  grabOrRelease: Grab or Release
  selectParticleInAtom: Select particle in atom
  moveGrabbedParticle: Move grabbed particle
  returnToBucket: Return grabbed particle to bucket
  cancelMovement: Cancel movement

# Preferences
preferences:
  chargeNotationSelectorLabel: "Choose sign convention for charges in chemical symbol."
```

Which produces nesting in the json:
<details>

```json
  "keyboardHelpContent": {
    "particleNavigationHeading": {
      "value": "Particle Navigation"
    },
    "grabOrRelease": {
      "value": "Grab or Release"
    },
    "selectParticleInAtom": {
      "value": "Select particle in atom"
    },
    "moveGrabbedParticle": {
      "value": "Move grabbed particle"
    },
    "returnToBucket": {
      "value": "Return grabbed particle to bucket"
    },
    "cancelMovement": {
      "value": "Cancel movement"
    }
  },
  "preferences": {
    "chargeNotationSelectorLabel": {
      "value": "Choose sign convention for charges in chemical symbol."
    }
  },
```
</details>

For #431

**Comment by jbphet** [comment]
Yes, nested strings are now supported by the live version of Rosetta.  See https://github.com/phetsims/rosetta/issues/460.

Also, just to increase my confidence in my answer, I looked at the "Chains" test sim using the live version of the translation utility, since it now contains a nested non-a11y string.  I verified that the nested string is correctly presented in the translation UI and that the submitted values for the `ab` (Abkhazian) locale look correct on both the built version and in the translation file, and they do.

Closing.
